src/cli.c

```diff
@@ -208,6 +222,23 @@ int main(int argc, char **argv) {
                 exit_code = EXIT_INVALID_ARG;
                 goto exit;
             }
+            if (opt.key == 'l' || is_longopt(opt, "load")) {
+                char *filepath = get_option_value(arg, argc, argv, &optind);
+                if (!filepath) {
+                    report_missing_argument(&opt);
+                    exit_code = EXIT_INVALID_ARG;
+                    goto exit;
+                }
+                FileItem *file = (FileItem *) (malloc(sizeof(FileItem)));
```

Comment:
No need to cast the result of malloc. Also, you can simplify this as follows:
`FileItem *file = malloc(sizeof(*file));`